test/unit/core/contentLookup.test.js

```diff
@@ -48,13 +48,22 @@ describe('Calling the getContentAsString() function', () => {
 describe('Calling the getContentAsArray() function', () => {
   it('should retrieve the content of all events as an array', () => {
     allEventKeys.forEach(key => {
-      const contentArr = getContentAsArray(`status.${key}.content`);
+      const contentArr = getContentAsArray(`status.${key}.oral.content`);
       return expect(contentArr).to.be.an('array').that.is.not.empty;
     });
   });
   it('should...', () => {
```

Comment:
These two tests should have clearer descriptions.

Reply:
True! Shall amend 👍

Reply:
👍